### PR TITLE
Update troubleshooting in README.md

### PR DESCRIPTION
**Pull request by RouNNdeL**
Adds a 'Devices found, but not initialized' section.

Addresses #197

### CODE REVIEW

**Review by KapJI — APPROVED** [review]
Looks good, thanks!

**Comment by DurgNomis-drol** [comment]
Shouldn't it be pointed out to the user in the installation fase instead? Maybe in the config flow?. 😄

File: README.md [hunk 1]
```diff
@@ -298,6 +298,12 @@ Here are the steps to resolve this issue:
    3. Paste the password into the **Google account app password** field.
    4. Click **Submit**.
 
+### Devices found, but not initialized
+
+If the debug logs list your devices, but then show `Successfully initialized 0 Google Home devices` make sure you are logged into the correct Google account.
+Your account has to be able to access the devices in your Home.
+If unsure, you can check what account you are using in the _Google Home_ app.
```
Comment (leikoilja) [inline README.md:305]:
```suggestion
The account you are using with the integration must have access to your google devices.
If unsure, you can check what account and account permissions in the mobile _Google Home_ app.
```

**Review by leikoilja — COMMENTED** [review]
Awesome! Thanks for contributing, @RouNNdeL 🚀

File: README.md (hunk 1 above)
Comment (KapJI) [reply README.md]: @leikoilja Sorry, I think the second part became worse :)

How about
```markdown
If unsure, please check what account you are using in the _Google Home_ app and if you have these devices listed there.
```

**Comment by RouNNdeL** [comment]
> Awesome! Thanks for contributing, @RouNNdeL

Thank you guys, for the awesome integration :smile:

File: README.md (hunk 1 above)
Comment (leikoilja) [reply README.md]: haha sound good, thanks for the spot

Comment (RouNNdeL) [reply README.md]: Yup, this sounds the best

Comment (RouNNdeL) [reply README.md]: Slightly change it, what do you guys think?

File: README.md [hunk 2]
```diff
@@ -298,6 +298,12 @@ Here are the steps to resolve this issue:
    3. Paste the password into the **Google account app password** field.
    4. Click **Submit**.
 
+### Devices found, but not initialized
+
+If the debug logs list your devices, but then show `Successfully initialized 0 Google Home devices` make sure you are logged into the correct Google account.
+The account you are using with the integration must have access to your Home.
+If unsure, please check what account you are using in the mobile _Google Home_ app and if your devices are listed there.
```
Comment (KapJI) [inline README.md:305]:
```suggestion
If unsure, please check what account you are using in the _Google Home_ app and if your devices are listed there.
```

**Comment by KapJI** [comment]
Thanks @RouNNdeL!

**Pull request merged**